src/server.rs

```diff
@@ -283,10 +285,78 @@ async fn upload(
     Ok(HttpResponse::Ok().body(urls.join("")))
 }
 
+/// File entry item for list endpoint.
+#[derive(Serialize, Deserialize)]
+pub struct ListItem {
+    /// Uploaded file name.
+    pub file_name: String,
+    /// Size of the file in bytes.
+    pub file_size: u64,
```

Comment:
I am wondering, whether the underscore is necessay for the first 2 items.

Isn't `filename` and `filesize` rather used in most situations?

Reply:
`file_name` gets used pretty frequently elsewhere in this source but it doesn't matter to me either way.

Reply:
right, but in this case it is public and the key name in the JSON output. e.g. if you were to use jq or any other tool to filter the output, IMO keys without a underscore would be easier to type. And fom filename looks better than file_name.
But that's subjective. I am just lazy when it comes to typing.

Reply:
Sounds logical to me. I went ahead and made this change.